package.json

```diff
@@ -75,7 +75,8 @@
 		"build-client-if-desktop": "check-npm-client && node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || yarn run build-client-fallback",
 		"build-client-stats": "check-npm-client && NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true PROGRESS=true yarn run build-client-evergreen",
 		"build-packages": "check-npm-client && npx lerna run prepare --stream",
-		"build-packages:watch": "check-npm-client && chokidar \"./packages/**\" -i \"./packages/*/dist/**\" -i \"**.css\" -i \"**.d.ts\" --debounce 1000 --throttle 10000 -c \"yarn run build-packages\"",
+		"build-packages:watch": "check-npm-client && chokidar \"./packages/**\" -i \"./packages/*/dist/**\" -i \"**.css\" -i \"**.d.ts\" -i \"**.md\" -i \"./packages/*/test/**\" -i \"./packages/*/tests/**\" --debounce 1000 --throttle 10000 -c \"yarn run build-packages:rebuild-one {path}\"",
```

Comment:
I know this was an existing problem, but I think it is important to point it out:

`build-packages` builds anything that has a `prepare` script. This includes some apps in `./apps`. I'd expect `build-packages:watch` to operate on the same set of packages, but in watch mode. 

However, the proposed implementation will limit it to `./packages` directory

Reply:
A quick, simple win is for apps to stop using `prepare`. I don't think there was justification to use that in the first place and they should use `build`.

Nothing should depend on apps, they shouldn't need to be built for the Calypso client to work.

Reply:
(Not for this PR)

Following that logic, there may be `./packages` that are not required for Calypso at all.

I wonder if we could drop `build-packages` as a prerequisite and let webpack handle the compilation/transpilation of the used packages (eg: use https://github.com/TypeStrong/ts-loader for TS packages).

Reply:
That is absolutely another direction to test, at the risk of blurring the line between Calypso client and the packages. We've tried to maintain strong isolation with packages, so the Calypso client is effectively just another consumer of the external packages even though they're in the same repo.

I'd love to see exploration in our project setup and TypeScript tooling in particular 🙂